notebooks/asian_barrier_option/deep_learning_option_1.ipynb

```diff
@@ -0,0 +1,850 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Deep Learning Barrier Option\n",
+    "\n",
+    "We used Numba and CuPy in the previous notebook to run Monte Carlo simulation to determine the price of the barrier option. A Monte Carlo simulation needs millions of paths to get an accurate answer which is computationally intensive. [Ryan et al (2018)](https://arxiv.org/abs/1809.02233) showed that a deep learning model can be trained to value derivatives. The deep learning model is accurate and very fast, capable of producing valuations a million times faster than traditional models. In the this notebook, we will use a fully connected network to learn the pricing mode of the Asian Barrier option. Monte Carlo simulation is used as pricing ground truth for the training. \n",
+    "\n",
```

Comment:
For the deep learning model, we don't use this specific option. We will generate random option parameters to train the model.

Reply:
I meant just re-list what the parameters stand for not their exact values.
```
T - Maturity (yrs.)
S - Spot (usd)
K - Strike (usd)
sigma - Volatility (per.)
r - Risk Free Rate (per.)
mu - Stock Drift Rate (per.)
B - Barrier (usd)
```

It would be helpful to have it repeated in the notebook so the notebook could be more self contained instead of having to look up what those parameters mean from the other notebooks ("mc_pricing.ipynb" in particular).

Reply:
okay. I will do this